cookbooks/bcpc/recipes/networking.rb

```diff
@@ -55,6 +55,26 @@
   action :remove
 end
 
+log 'udev persistent-net.rules file found! Your NICs may have been renamed.' do
+  level :warn
+  only_if { File.exists?('/etc/udev/rules.d/70-persistent-nic.rules') }
+end
+
+#
+# Disable NIC renaming on future boots by creating an empty
+# persistent-net-generator to override the one in /lib/udev
+#
+file '/etc/udev/rules.d/75-persistent-net-generator.rules' do
+  content "# This file was created by Chef.\n" \
+          "# (It is intentionally empty.)\n"
```

Comment:
@http-418 You're usually really consistent on these things but any reason `"` is needed here instead of `'`?